beaconChain/lib/forkChoice/lmdGhost.ts

```diff
@@ -0,0 +1,100 @@
+import {Attestation, BeaconBlock, BeaconState, PendingAttestation, Validator, ValidatorIndex} from "../../types";
+import {getActiveValidatorIndices, getEffectiveBalance, slotToEpoch} from "../../helpers/stateTransitionHelpers";
+import {FORK_CHOICE_BALANCE_INCREMENT} from "../../constants";
+import {Slot} from "../../types";
+
+type int = number;
+
+// Probably add this as a field to BeaconState
+interface Store {
+  blocks: BeaconBlock[];
+  pendingAttestations: PendingAttestation[];
+  validatorRegistry: Validator[];
```

Comment:
I don't think validatorRegistry should be here, validation is done from the perspective of a single validator.